shared/constants/chat2/meta.js

```diff
@@ -452,3 +452,10 @@ export const getConversationRetentionPolicy = (
 
 export const isDecryptingSnippet = (meta: Types.ConversationMeta) =>
   meta.trustedState === 'requesting' || meta.trustedState === 'untrusted'
+
+export const getTeams = (state: TypedState): Array<string> => {
+  const teamMap = state.chat2.metaMap.reduce((m, meta) => {
+    return meta.teamname ? m.set(meta.teamname, true) : m
```

Comment:
Just pass an array to reduce and push

Reply:
This is to de-dup the names, since you can have multiple convs with same team name.

Reply:
Can't you just use general

Reply:
Ah yeah good point